### PR TITLE
GRPC and Protobuffer documentation

### PR DESCRIPTION
**Pull request by nityanandaz**
## Problem that is solved
GRPC and Protobuffer exporters get a visionary document

### CODE REVIEW

**Review by moritzschuell — COMMENTED** [review]
Thanks for moving forward with a PR for the docu @nityanandaz!
This branch has been lying around quite some time 😁

I reviewed & fixed some typos. But since I was also involved with this docu, we should maybe request a review from Apodini/apodini before merging 🤔

File: Documentation/AdditionalFunctionality/Protocol Buffers.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+```swift
+var content: some Component {
+    Group("greet") {
+        Greeter()
+    }
+}
+```
+becomes
+```proto
+service V1GreeterService {
```
Comment (theMomax) [inline Documentation/AdditionalFunctionality/Protocol Buffers.md]: Should this be V1GreetService?

**Review by theMomax — COMMENTED** [review]
Looks good, but is this really just a vision? You could consider marking sections that are already implemented with the documentation label below the according headline.

File: Documentation/AdditionalFunctionality/Protocol Buffers.md (hunk 1 above)
Comment (nityanandaz) [reply Documentation/AdditionalFunctionality/Protocol Buffers.md]: Yes 🙂

**Review by theMomax — APPROVED** [review]
LGTM👍🏻

**Pull request merged**